imap_processing/cdf/utils.py

```diff
@@ -63,7 +63,10 @@ def write_cdf(
     file_start_date = None
     if "idex" in data.attrs["Logical_source"]:
         file_start_date = data["Epoch"][0].data
-    elif "swe" in data.attrs["Logical_source"]:
+    elif (
```

Comment:
Is there a good reason we are inconsistent here, or can we somehow make this consistent across the instruments?
Possibly an `else:` for now though?

Reply:
Good question, I am not sure of the answer. Perhaps @tech3371 and/or @bryan-harter has some insight here?